packages/fx-core/src/plugins/resource/sql/managementClient.ts

```diff
@@ -11,6 +11,7 @@ export class ManagementClient {
   client: SqlManagementClient;
   config: SqlConfig;
   ctx: PluginContext;
+  nextIndex = 0;
```

Comment:
I may suggest better variable naming here, e.g. totalFirewallRuleCount or something else

Reply:
It is fixed. Thanks.